src/app/views/query/table/query-table.component.js

```diff
@@ -68,9 +68,12 @@ class QueryTableController {
   }
 
   getTableColumns() {
-    return this.tableColumns.map(column =>
-      typeof column === 'string' ? column : column.column,
-    );
+    if (this.tableColumns) {
+      return this.tableColumns.map(column =>
+        typeof column === 'string' ? column : column.column,
```

Comment:
why would tableColumns sometimes be a list of objects & sometimes a list of strings? I think we should fix that issue and then remove this map function.

Reply:
when querying entity 'fastq', table column 'BioSample Accession #' is returned as an object

Reply:
I'm assuming you did it that way because the results object returns BioSample as an object, whereas all the others are strings. 
Lmk if you want me to edit the query.service, so that BioSample just returns the accession # as a string, or if you want to keep it the way it is